src/main/java/com/google/api/codegen/config/LanguageGenerator.java

```diff
@@ -132,6 +135,23 @@ public String getFormattedPackageName(String packageName) {
     }
   }
 
+  private static class NodeJSLanguageFormatter extends SimpleLanguageFormatter {
```

Comment:
Don't extend SimpleLanguageFormatter - you're overriding the only method defined. Just use "implements LanguageFormatter".